docs/08-resources.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Assets
+path: /docs/assets
+---
+
+Almost every game is more than just boxes and circles. Excalibur provides an asset loading feature that can help you preload things like JSON files, images, and sounds.
+
+## Creating an asset loader
+
+When calling [[Engine.start]], you can optionally pass an asset [[Loader]]. This loader will contain a reference to any [[Loadable|"loadables"]] you want to load.
+
+```ts
+const loader = new ex.Loader([
+  /* add Loadables here */
+])
+```
+
+[[Loadable|Loadables]] are different kinds of assets such as textures, sounds, and generic resources.
+
+<docs-note>**Anytime** you call `game.start(loader)`, the game will pause and the engine will load assets. This means that you _do not have to load every asset at once_! Instead you may want to call `game.start(loader)` initially with core assets and then again when [initializing a Scene](/docs/scenes#initialization).</docs-note>
+
+## Using a web server
+
+The asset loader **only works with a web server** since it loads assets with [XHR](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest). That means you cannot use the loader when double-clicking and running an HTML file in the browser. The browser throws errors that will prevent you from loading assets.
+
+The fastest way to serve a folder of files is by using the [serve](https://npmjs.org/package/serve) NPM package.
+
+```bash
+# Serve the current directory
+npx serve .
+
+# Serve a folder
+npx serve ./dist
+```
+
+If you are developing a game using Excalibur with Webpack, Parcel, or another bundler, these typically already come with dev servers for running your game.
```

Comment:
Some links to template repos for each of the bundler mentioned might be good here 😃